P/PETSc/build_tarballs.jl

```diff
@@ -120,16 +156,22 @@ augment_platform_block = """
 
 # We attempt to build for all defined platforms
 platforms = expand_gfortran_versions(supported_platforms(exclude=[Platform("i686", "windows")]))
-
 platforms, platform_dependencies = MPI.augment_platforms(platforms)
 
 # Avoid platforms where the MPI implementation isn't supported
 # OpenMPI
 platforms = filter(p -> !(p["mpi"] == "openmpi" && arch(p) == "armv6l" && libc(p) == "glibc"), platforms)
+#platforms = filter(p -> !(p["mpi"] == "openmpi" && os(p) == "linux"), platforms)  
+#platforms = filter(p -> !(p["mpi"] == "openmpi" &&  Sys.isfreebsd(p)), platforms)  
+
 # MPItrampoline
 platforms = filter(p -> !(p["mpi"] == "mpitrampoline" && libc(p) == "musl"), platforms)
 platforms = filter(p -> !(p["mpi"] == "mpitrampoline" && Sys.isfreebsd(p)), platforms)
+platforms = filter(p -> !(p["mpi"] == "mpitrampoline" && arch(p) == "armv7l"), platforms)
+platforms = filter(p -> !(p["mpi"] == "mpitrampoline" &&   os(p) == "linux"), platforms)
+
 
+# Not working @ the moment is linux + openmpmi
```

Comment:
Is this comment accurate?

Reply:
Not anymore as the tests are running fine now

Reply:
Can we remove it then?

Reply:
sure, I'll do that as part of implementing the other requests of @eschnett